sap_business_document_processing/document_information_extraction_client/dox_api_client.py

```diff
@@ -178,7 +180,8 @@ def extract_information_from_document(self, document_path: str, client_id, docum
         :param document_path: The path to the document
         :param client_id: The client ID for which the document should be uploaded
         :param document_type: The type of the document being uploaded. For available document types see documentation
-        :param mime_type: Content type of the uploaded file. If None is given, the content type is fetched automatically.
+        :param mime_type: Content type of the uploaded file. If 'unknown' is given, the content type is fetched
+        automatically. Default is 'application/pdf'.
```

Comment:
Should we extend the docstring to hint to the constants that can be used to pass in this information?